hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/s3guard.md

```diff
@@ -1474,6 +1474,18 @@ Caused by: java.lang.NullPointerException
   ... 1 more
 ```
 
+### Error `Attempt to change a resource which is still in use: Table is being deleted`
```

Comment:
as I see this is an entirely different issue, which should be resolved in a separate commit. do you plan to do that instead of squashing all your changes to the same commit?

Reply:
I've been working on this patch for too long and wrapping up stuff to try and get all tests to worse; a lot of work has gone into the ITestDynamoDB there. I don't want to split them up at this late in the patch. Sorry

Reply:
Worth discussing I think--I'm with @bgaborg on this. I've been on projects in the past that auto-reject conflated commits. Why not maintain a list of commits on your branch and commit them intact instead of squashing them? Git rebase -i and force push (your personal branch only) are your friends here. Gives you optimal commit history and not that hard to manage IMO. Maybe try it out next time you are working on a patch set. I wouldn't force you to go break apart these commits this late in the game, though.

Reply:
I do have branches of the unsquashed PRs, so I should be able to do that. By popular requeset I will give it a go, but leave them in here. If people get the other PR in first, I'll deal with that